# Исключения и оброботка ошибок
# print(name)
# try:
#     # print(name)
#     list1 = [1, 2, 3]
#     print(list1[5])
# # except Exception as e:  # Что надо делать если произошло ошибка
# except (NameError, IndexError) as e:  # Что надо делать если произошло ошибка
#     e.__str__ = "ошибка, такой переменной нет"
#     print('Произошло ошибка', e)
#     print('Произошло ошибка', e.__class__)
#     print('Произошло ошибка', e.__class__.__name__)
#     print('Произошло ошибка', e.__str__)


# Бесконечно спрашиваем у пользователя номер
# Если введеннный текст не оказался числом - продолжить работу но вывести текст ошибки
# Если число == 1 - распечатать name которого не существует-вывести текст ошибки
# Если число == 2 - вывести 6 элемент из списка (из 3х элементов)-вывести имя ошибки
# Если число == 3 - попробовать разделить 10/0 - вывести свой текст ошибки